slam_toolbox:
  ros__parameters:
    use_sim_time: True
    odom_frame: "odom"
    base_frame: "base_footprint"
    scan_topic: "scan"
    map_frame: "map"
    minimum_travel_distance: 0.3
    minimum_travel_heading: 0.3
    resolution: 0.05
    map_update_interval: 1.0       # Update the map every 1 second
    publish_period: 1.0            # Publish the map every 1 second